Library/Formula/ecl.rb

```diff
@@ -13,4 +13,13 @@ def install
     system "make"
     system "make install"
   end
+
+  test do
+    (testpath/'simple.cl').write <<-EOS.undent
+      (write-line (write-to-string (+ 2 2)))
+    EOS
+    output = `'#{bin}/ecl' -shell #{testpath}/simple.cl`
+    assert_equal '4', output.strip
+    assert_equal 0, $?.exitstatus
```

Comment:
`assert_equal "4\n", shell_output("#{bin}/ecl -shell #{testpath}/simple.cl")` will do the same as the above three lines.